src/database/database.go

```diff
@@ -0,0 +1,186 @@
+package database
```

Comment:
The Go convention for putting packages in a subdir of the repo is `pkg`, not `src`. Please rename the dir accordingly.